pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -543,6 +552,11 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.mesonOption "mount-path" "${lib.getOutput "mount" util-linux}/bin/mount")
     (lib.mesonOption "umount-path" "${lib.getOutput "mount" util-linux}/bin/umount")
 
+    # SSH
+    # Disabled for now until someone makes this work.
+    (lib.mesonOption "sshconfdir" "no")
+    (lib.mesonOption "sshdconfdir" "no")
```

Comment:
# 👀 
[Interesting feature](https://www.freedesktop.org/software/systemd/man/devel/systemd-ssh-proxy.html).

Reply:
You can have the files with a small patch but I am not sure that they make sense without the accompanying VM setup. They ought to be linked into the system's /etc.

```patch
diff --git pkgs/os-specific/linux/systemd/default.nix pkgs/os-specific/linux/systemd/default.nix
index 09739968e55d..60be69acb602 100644
--- pkgs/os-specific/linux/systemd/default.nix
+++ pkgs/os-specific/linux/systemd/default.nix
@@ -554,8 +554,8 @@ stdenv.mkDerivation (finalAttrs: {

     # SSH
     # Disabled for now until someone makes this work.
-    (lib.mesonOption "sshconfdir" "no")
-    (lib.mesonOption "sshdconfdir" "no")
+    (lib.mesonOption "sshconfdir" "${placeholder "out"}/etc/ssh")
+    (lib.mesonOption "sshdconfdir" "${placeholder "out"}/etc/ssh")

```